src/witness/witness.ml

```diff
@@ -505,14 +505,14 @@ struct
         in
         (module TaskResult:WitnessTaskResult)
       )
-    | ValidFree ->
-      let module TrivialArg =
+    | ValidFree (*->*)
```

Comment:
Before merging this, we'll need to cleanup the commented out code.

Reply:
Should be cleaned up now (https://github.com/goblint/analyzer/pull/1201/commits/a975702f700511c3c1f0b71979502462192d9bc2)